modules/logging-rn-5.7.1.adoc

```diff
@@ -0,0 +1,27 @@
+//module included in logging-release-notes.adoc
+:_content-type: REFERENCE
+[id="logging-release-notes-5-7-1_{context}"]
+= Logging 5.7.1
+This release includes: link:https://access.redhat.com/errata/RHBA-2023:3197[OpenShift Logging Bug Fix Release 5.7.1].
+
+[id="logging-5-7-1-bug-fixes"]
+== Bug fixes
+* Before this update, the presence of numerous noisy messages within Cluster Logging Operator pod logs caused reduced log readability and increased difficulty in identifying important system events. With this update, the issue is resolved by significantly reducing the noisy messages within Cluster Logging Operator pod logs, resulting in improved log readability and enhanced visibility of critical system events. (link:https://issues.redhat.com/browse/LOG-3482[LOG-3482])
+
+* Before this update, the API server would default the new collector type field to "vector" even when the custom resource had the prior field selected. This update removes the default for the new collector type field to restore the previous behavior. (link:https://issues.redhat.com/browse/LOG-4086[LOG-4086])
+
+* Before this update, within the {Product-Title} web console a time range could not be selected by click dragging over the logs histogram. With this update, click dragging can successfully select a time range. (link:https://issues.redhat.com/browse/LOG-4501[LOG-4501])
+
+* Before this update, the action of clicking on the *Show Resources* link within the {Product-Title} web console did not produce any effect. With this update, the issue is resolved by fixing the functionality of the "Show Resources" link to toggle the display of resources for each log entry. (link:https://issues.redhat.com/browse/LOG-3218[LOG-3218])
+
+
+[id="logging-5-7-1-CVEs"]
```

Comment:
all module IDs need to include context per https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-in-module-files